### ISSUE TITLE
feat(accessibility): update IconButton components to be labeled for using screen readers

### ISSUE THREAD

**Issue by scarletmerlin123** [OpenCTI-Platform/opencti#16886]
## Use case

Many of the IconButton components lack labels and are not able to be read properly by screen readers 

## Current workaround

None

## Proposed solution

Update IconButton components that are missing aria-labels

## If the feature request is approved, would you be willing to submit a PR?

Yes

**Comment by romain-filigran** [comment]
Related to https://github.com/OpenCTI-Platform/opencti/issues/6612
